lib/view/list_preview_widget.dart

```diff
@@ -37,8 +37,26 @@ class _ListPreviewWidgetState extends State<ListPreviewWidget> {
         setState(() {});
       },
       onLongPress: _showOptionsModalSheet,
-      title: Text(widget.listModel.title,
-          style: Theme.of(context).textTheme.titleLarge),
+      title: Row(
```

Comment:
No need to create a `Row` here in the `Title` slot. Just put your `Wrap` in the `trailing` slot of the `ListTile`

Reply:
Well, I was having some trouble with putting the `Wrap` in the `trailing` slot. The labels would not wrap until they hit the far edge of the `ListTile`, forcing the title to softwrap like mad.

here are some screenshots:



![titlewrapproblem](https://github.com/Pertempto/Lists/assets/91034018/65dbc95c-b75f-41f7-865c-5df2fe97cb93)



![evenmoretitlewrapping](https://github.com/Pertempto/Lists/assets/91034018/bfaf7b2a-7a4c-400a-8869-ea3c5eb66a6d)

Reply:
hahaha whoa that is a problem.

Reply:
In this case, we are running up against the limits of what `ListTile` can do. I pushed a commit that replaces it with a custom `InkWell`. I used a `Container` to force the labels to stay in a limited width, so it looks better on mobile. Make any adjustments that you need.